config/routes.rb

```diff
@@ -32,11 +32,11 @@
         as: :voice_otp,
         defaults: { format: :xml }
 
+  post '/api/usps_upload' => 'usps_upload#create'
```

Comment:
My preference would be to have a `/v1/` in there so we can version endpoints if we need to. Helps us avoid at the api level what we ran into with routes changing between deploys.

Reply:
I also see that we haven't done versioning for any of our other endpoints, so this should be okay. We can always use `/v1/` when we create the next version of any of these.